tools/build-android.sh

```diff
@@ -54,6 +56,6 @@ for bt in "${BUILD_TYPES[@]}"; do
     done
 done
 
-v=$(git describe --tags)
+v=v$(grep "^VERSION=" $REALM_CORE_DIR/dependencies.list | cut -d '=' -f2)
```

Comment:
`$REALM_CORE_DIR/dependencies.list` needs quotes to not break if the source path has spaces.